DemoApp/StreamChat/Extensions/DeletedMessageVisibility+CustomStringConvertible.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+extension ChatClientConfig.DeletedMessageVisibility: CustomStringConvertible {
```

Comment:
What if we just make the change in LLC? When this setting is moved to the backend, string would be the way to encode/decode it from JSON so it's fine to make this change, I think
```swift
public enum DeletedMessageVisibility: String {
        /// All deleted messages are always hidden.
        case alwaysHidden
        /// Deleted message by current user are visible, other deleted messages are hidden.
        case visibleForCurrentUser
        /// Deleted messages are always visible.
        case alwaysVisible
    }
```

Reply:
Im happy to do so, if there are no objections from the team.

Reply:
This small bit in our demo app pinpoints another deficiency in our LLC layer. 
Our models aren't `CustomDebugStringConvertible`, thus they can't be nicely interpreted by the LLDB.

Reply:
I moved that conformance to LLC